### PR TITLE
Added frontend for mindspore.ops.hardswish

### LINKED ISSUE #21274: hardswish
Frontend implementation for mindspore.ops.hardswish

### PR DESCRIPTION
**Pull request by yashj8**
Close #21274

### CODE REVIEW

**Comment by yashj8** [comment]
@xoiga123 
Resolved merge conflicts
Lint is failing on line 105 which is not part of my code. It is part of the pad function

File: ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py [hunk 1]
```diff
@@ -79,6 +79,18 @@
...
+#      fn_tree="mindspore.ops.function.nn_func.hardswish",
+#      dtype_and_x=helpers.dtype_and_values(
+#          available_dtypes=helpers.get_dtypes("valid"),
+#      ),
+# )
+# def test_mindspore_hardswish(
+#     *,
+#     dtype_and_x,
+
+
```
Comment (xoiga123) [inline ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py:93]: Please finish the test as well :sweat_smile:

**Review by xoiga123 — CHANGES_REQUESTED** [review]
Hi @yashj8, sorry for the late reply. Thank you for contributing to Ivy :star: The implementation seems good but maybe you haven't finished the test?

File: ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py (hunk 1 above)
Comment (yashj8) [reply ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py:93]: Might've messed up during resolving merge conflict😅 Thanks for the feedback. Resolving it

**Comment by yashj8** [comment]
Hi @xoiga123! 
I have restored the code for the test
Kindly take a look and let me know of any other requirements 😄 
Thanks!

File: ivy/functional/frontends/mindspore/ops/function/nn_func.py [hunk 2]
```diff
@@ -38,6 +38,25 @@ def softsign(x):
...
+    {
+        "2.0 and below": (
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "float16",
+            "float32",
+            "float64",
+        )
```
Comment (xoiga123) [inline ivy/functional/frontends/mindspore/ops/function/nn_func.py:51]: Sorry I haven't noticed this before in the earlier review, but where did you find this info? It sounds correct but I can't seem to find it anywhere in their docs or code...

Comment (xoiga123) [reply ivy/functional/frontends/mindspore/ops/function/nn_func.py:51]: Got it, thank you!

Comment (yashj8) [reply ivy/functional/frontends/mindspore/ops/function/nn_func.py:51]: Please let me know of any further requirements 😄

**Comment by yashj8** [comment]
Hello @satakshigarg! Can you please take a look at it and let me know of any required changes? This PR is awaiting review from you for a while.
Thanks 😄

**Comment by satakshigarg** [comment]
Thanks @yashj8 for your contributions !!
It looks good to me !!

Keep contributing 😃

**Comment by yashj8** [comment]
Thanks a lot @satakshigarg !!😄 
Feels great to have my first PR merged!

Looking forward to many more down the line..

**Pull request merged**